app/views/admin/users/index.html.erb

```diff
@@ -0,0 +1,15 @@
+<h1>Welcome to CMS. You are logged in as an Admin</h1>
+<table>
+  <tr>
+    <th>First Name</th>
+    <th>Middle Name</th>
+    <th>Last Name</th>
+    <th>country</th>
+    <th>Phone Number</th>
+    <th>Email</th>
+    <th colspan="3" ></th>
```

Comment:
Extra space?

Reply:
Faisal bhai as you can see these are the nested tags therefore the appropriate 2-space tabs are added.

<img width="988" alt="Screenshot 2021-04-14 at 3 05 49 PM" src="https://user-images.githubusercontent.com/79962712/114693317-e7ee1e80-9d32-11eb-88d5-f4075cdc5807.png">

Reply:
Extra space is after `"3"`